src/test/java/cloud/fogbow/ras/core/OrderControllerTest.java

```diff
@@ -211,26 +425,20 @@ public void testGetResourceInstanceNullOrder() throws Exception {
         this.ordersController.getResourceInstance(null);
     }
 
-    // test case: Tests if getUserAllocation() returns the ComputeAllocation properly.
+    // test case: Tests if the getUserAllocation method returns the ComputeAllocation properly.
```

Comment:
I think we should test a more complex case ... one that add up at least two allocations for the same user

Reply:
Okay, I'll change the test.